### ISSUE TITLE
glibc version

### ISSUE THREAD

**Issue by Schmebulock** [kellnr/kellnr#372]
Is there any reason, why glibc 2.38 is required? I'm running Kellnr on Ubuntu 22.04 LTS, today I wanted to upgrade Kellnr to the latest version and I'm getting 

```
./kellnr: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.38' not found (required by ./kellnr)
./kellnr: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.38' not found (required by ./kellnr)
```

Unfortunately on Ubuntu 22.04 only glibc 2.35 is available and I really want neither to upgrade OS nor compile glibc.

**Comment by makavity** [comment]
It is because it is built with ubuntu 24.04 [(fixing errors for docker)](https://github.com/kellnr/kellnr/issues/313)
https://github.com/kellnr/kellnr/blob/main/.github/workflows/ci.yaml

I think, the easiest way is to build binary yourself

**Comment by Schmebulock** [comment]
How do I set the version string?
I cloned the repository, switched to tag v5.2.3, compiled everything, but when I run it, I see the version is 0.0.0-debug. I would like to see the real version (so I don't forget what I'm running).
![obrazek](https://github.com/kellnr/kellnr/assets/21037263/2ef8d4b2-ecc4-4275-8389-56581fbb46ba)

Related to this. When I start Kellnr for the first time (the same it is with the precompiled binaries) it shows version unknown:
![obrazek](https://github.com/kellnr/kellnr/assets/21037263/aadf049e-f8db-413c-afd2-4e82ac1f94b0)
